docs/database/schema-alternative.sql

```diff
@@ -0,0 +1,101 @@
+--
```

Comment:
I believe that `action_property` and `connector_property` might look incidentally the same but they are different _things_. E.g if one would like to add a flag whether a property is hidden or not, this could go only to `connector_property` (as I suppose that the action properties all have business meaning, and should be changeable by the cit user ?). Even if I wrong with my assumption one has still this extra degree of freedom to evolve these parts differently. 

Another, more technical argument is, that SQL debugging via relationship tables is always tedious, requiring two joins. Also you would have to add extra constraints to ensure that one config property only belongs to an action or a connector. And finally, looking in to this combine configuration_property table only, you can not decide whether a property is connector or action related. You would need to examine both relationship tables. And finally finally diamond structures are always evil ;-)